ui/src/views/Connectors.vue

```diff
@@ -680,8 +674,7 @@ export default {
     },
     openEditConnector(conn, email) {
       this.newConnector = Object.assign({}, conn);
-      this.newConnector.props.FilterCheck =
-        this.newConnector.props.FilterCheck == "enabled";
+      this.newConnector.props.FilterCheck = this.newConnector.props.FilterCheck == "enabled";
```

Comment:
Please avoid commits only for the code indentation because they really create a bad diff which is hard to review.
If a code indentation fix is really needed, move it to a different commit.